### PR TITLE
filled in nouns, actions, and psuedo code

### PR DESCRIPTION
**Pull request by MasonSelf**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-
-//  action 3:
- 
+//  Noun:       politician
+//  action 1:   the politician lies
+politician.lie();
+//  action 2:   the politician leads
+politician.lead();
+//  action 3:   the politician delivers a speech
+politician.deliverSpeech();
```
Comment (matkatmusic) [inline main.cpp:117]: As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

**Review by matkatmusic — CHANGES_REQUESTED** [review]

**Review by MasonSelf — COMMENTED** [review]
Here are the requested changes to example 8.

File: main.cpp [hunk 2]
```diff
@@ -108,13 +108,13 @@ drum.rollDownHill();
...
-politician.lie();
-//  action 2:   the politician leads
-politician.lead();
-//  action 3:   the politician delivers a speech
-politician.deliverSpeech();
+//  Noun:       coolCamperVan
+//  action 1:   the cool camper van pops its top
+coolCamperVan.popTop();
+//  action 2:   the cool camper van gets a flat tire
+coolCamperVan.getFlatTire();
```
Comment (matkatmusic) [inline main.cpp:115]: does that mean its no longer very cool? :-P

**Review by matkatmusic — APPROVED** [review]

File: main.cpp (hunk 2 above)
Comment (MasonSelf) [reply main.cpp:115]: It's all a part of the journey, but I never feel like it's cool in the moment.

**Pull request merged**